src/app/Mappings/Network/NetworkMappingsPage.tsx

```diff
@@ -16,12 +16,23 @@ import { PlusCircleIcon } from '@patternfly/react-icons';
 import { INetworkMapping, MappingType } from '../types';
 import MappingsTable from '../components/MappingsTable';
 import AddEditMappingModal from '../components/AddEditMappingModal';
+import { fetchMockStorage } from '../mocks/helpers';
 
 // TODO replace these with real state e.g. from redux
 const isFetchingInitialNetworkMappings = false; // Fetching for the first time, not polling
-const networkMappings: INetworkMapping[] = [];
+// const networkMappings: INetworkMapping[] = [];
 
 const NetworkMappingsPage: React.FunctionComponent = () => {
+  const [networkMappings, setNetworkMappings] = React.useState([]);
+
+  //TODO replace with real state from redux
+  const mockMapObj = localStorage.getItem('networkMappingsObject');
+  React.useEffect(() => {
```

Comment:
So I'm not sure I get the useEffect code here, why does it have `mockMapObj` in its dependency array? That won't change here I don't think, so the useEffect will only be triggered once. I think we can get rid of the `mockMapObj` and have `[]` as the deps array for the effect?

Reply:
The mockMapObj is required in the dependency array because when I added an item to the local storage, there was no rerender triggered. This will for sure go away when we move to real redux state but it is needed temporarily for the mock localstorage code to work for now.

Reply:
got it. ok.